blogster/config/database.yml

```diff
@@ -22,4 +22,4 @@ test:
 
 production:
   <<: *default
-  database: db/production.sqlite3
+  database: db/production.pg
```

Comment:
acredito que está faltando algumas conf: segue as conf que usamos


default: &default
  adapter: postgresql
  encoding: utf8
  pool: 5
  timeout: 5000

development:
  <<: *default
  database: blog_development

# Warning: The database defined as "test" will be erased and
# re-generated from your development database when you run "rake".
# Do not set this db to the same as development or production.
test:
  <<: *default
  database: blog_test

production:
  <<: *default
  database: blog_production

Reply:
Ótimo, alterei essas configurações por conta de um artigo que vi explicando o deploy de rails pro heroku. Vou complementar com essas configs